command/s3_put.go

```diff
@@ -214,6 +223,9 @@ retryLoop:
 			return errors.New("s3 put operation canceled")
 		case <-timer.C:
 			filesList := []string{s3pc.LocalFile}
+			// reset to avoid duplicated uploaded references
+
+			uploadedFiles = []string{}
```

Comment:
Move above 238, and delete space between it and its comment.

Reply:
aye aye